tests/thread_local_tests/README.md

```diff
@@ -0,0 +1,29 @@
+# Thread_local Tests Suite
+
+This test suite is used to detect whether AppVerifier will report a memory leak caused by thread_local.
+
+## Getting Started
+
+Thread_local tests are based on the googletest framework. You can filter tests with
+`--gtest_filter` and explore tests available with `--gtest_list_tests` options.
+
+### Pre-requisites
+
+- Windows OS to build the tests.
```

Comment:
is it possible to run such tests in GHA?

Reply:
this will be done in https://jira.devtools.intel.com/browse/CVS-128825.